src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -121,7 +132,7 @@ export class MatProgressSpinner extends _MatProgressSpinnerMixinBase implements
   }
   set value(newValue: number) {
     if (newValue != null && this.mode === 'determinate') {
-      this._value = Math.max(0, Math.min(100, newValue));
+      this._value = Math.max(0, Math.min(100, coerceNumberProperty(newValue)));
```

Comment:
We created a `clamp` method in `MdProgressBar` for this, should we do similar here?  Potentially moving clamp to a util?

If so it might be in a follow up PR, but thought I would throw it out there.

Reply:
We used to have a clamp function because it was used in a couple of places, but now it's only used once so it should be fine having it inline.